packages/x-tailwindcss/demo/src/components/xds-button.vue

```diff
@@ -11,7 +11,10 @@
       title="Click me to copy CSS classes"
       :disabled="section === 'Disabled'"
     >
-      {{ removeClassPrefix(cssClass, base) || 'button' }}
+      <CuratedIcon class="x-icon" />
```

Comment:
Be careful, if you click the icon you will get the 'x-icon' class

Reply:
not anymore